Tests/test_imagepath.py

```diff
@@ -152,7 +152,7 @@ def test_transform():
 
 def test_transform_with_wrap():
     # Arrange
-    p = ImagePath.Path([0, 1, 2, 3])
+    p = ImagePath.Path((0, 1, 2, 3))
```

Comment:
I'm less keen on this. Here, you're not just changing the code to setup the tests, you're actually changing what is being tested.

I know you might assume that tuples and lists should be interchangeable as arguments to Pillow code, but we've had issues like #3738

Reply:
Those are still being tested here: https://github.com/python-pillow/Pillow/blob/bf00a4534d728dff93232142cf99aabe4e8174fd/Tests/test_imagepath.py#L42-L58
I didn't change every list to a tuple.

Reply:
I've added more tests with c090fe1.

Reply:
I've created https://github.com/Yay295/Pillow/pull/3